docs/pullsubscription/README.md

```diff
@@ -191,7 +198,7 @@ If you are replaced `TOPIC_NAME` manually, then make sure you delete the
 resulting YAML:
```

Comment:
This sentence originally does not make sense. maybe 

`then make sure you delete the  resulting YAML`

becomes

`then you can use the updated files to delete using:`